docs/comptime_jeremie.org

```diff
@@ -0,0 +1,32 @@
+** comptime
+As the name implies, the `comptime` keyword is used to evaluate an expression at compile time. This is useful when you want to ensure that a certain value is known at compile time, or when you want to generate code based on a compile-time value.
+
+*** When to use 'comptime'
+**** Assuring a value is known at compile time
+When you want to ensure that a certain value is known at compile time (e.g. the size of an array)
+
+**** Performance
+To assure the compiler that he can do certains things at compilation time in order to optimize performances by not doing certain things at runtime. (if the compiler gives you the power to do things at compile time so easily, you should use it)
```

Comment:
Maybe the first sentence is a bit complex to read (I need to read 2x to understand it). Refactor?

Reply:
done